src/devices/cpu/sh2/sh2dasm.cpp

```diff
@@ -15,596 +15,600 @@ static const char *const regname[16] = {
 
-static uint32_t op1101(char *buffer, uint32_t pc, uint16_t opcode)
+static uint32_t op1101(std::ostream &stream, uint32_t pc, uint16_t opcode)
 {
-	sprintf(buffer, "MOV.L   @($%02X,PC),%s [%08X]", (opcode * 4) & 0xff, regname[Rn], ((pc + 2) & ~3) + (opcode & 0xff) * 4);
+	util::stream_format(stream, "MOV.L   @($%02X,PC),%s [%08X]", (opcode * 4) & 0xff, regname[Rn], ((pc + 2) & ~3) + (opcode & 0xff) * 4);
 	return 0;
 }
 
-static uint32_t op1110(char *buffer, uint32_t pc, uint16_t opcode)
+static uint32_t op1110(std::ostream &stream, uint32_t pc, uint16_t opcode)
 {
-	sprintf(buffer, "MOV     #$%02X,%s", (opcode & 0xff), regname[Rn]);
+	util::stream_format(stream, "MOV     #$%02X,%s", (opcode & 0xff), regname[Rn]);
 	return 0;
 }
 
-static uint32_t op1111(char *buffer, uint32_t pc, uint16_t opcode)
+static uint32_t op1111(std::ostream &stream, uint32_t pc, uint16_t opcode)
 {
-	sprintf(buffer, "unknown $%04X", opcode);
+	util::stream_format(stream, "unknown $%04X", opcode);
 	return 0;
 }
 
-unsigned DasmSH2(char *buffer, unsigned pc, uint16_t opcode)
+unsigned DasmSH2(std::ostream &stream, unsigned pc, uint16_t opcode)
 {
 	uint32_t flags;
 
 	pc += 2;
 
-	switch((opcode >> 12) & 15)
+	switch ((opcode >> 12) & 15)
 	{
-	case  0: flags = op0000(buffer,pc,opcode);    break;
-	case  1: flags = op0001(buffer,pc,opcode);    break;
-	case  2: flags = op0010(buffer,pc,opcode);    break;
-	case  3: flags = op0011(buffer,pc,opcode);    break;
-	case  4: flags = op0100(buffer,pc,opcode);    break;
-	case  5: flags = op0101(buffer,pc,opcode);    break;
-	case  6: flags = op0110(buffer,pc,opcode);    break;
-	case  7: flags = op0111(buffer,pc,opcode);    break;
-	case  8: flags = op1000(buffer,pc,opcode);    break;
-	case  9: flags = op1001(buffer,pc,opcode);    break;
-	case 10: flags = op1010(buffer,pc,opcode);    break;
-	case 11: flags = op1011(buffer,pc,opcode);    break;
-	case 12: flags = op1100(buffer,pc,opcode);    break;
-	case 13: flags = op1101(buffer,pc,opcode);    break;
-	case 14: flags = op1110(buffer,pc,opcode);    break;
-	default: flags = op1111(buffer,pc,opcode);    break;
+		case  0: flags = op0000(stream, pc, opcode); break;
+		case  1: flags = op0001(stream, pc, opcode); break;
+		case  2: flags = op0010(stream, pc, opcode); break;
+		case  3: flags = op0011(stream, pc, opcode); break;
+		case  4: flags = op0100(stream, pc, opcode); break;
+		case  5: flags = op0101(stream, pc, opcode); break;
+		case  6: flags = op0110(stream, pc, opcode); break;
+		case  7: flags = op0111(stream, pc, opcode); break;
+		case  8: flags = op1000(stream, pc, opcode); break;
+		case  9: flags = op1001(stream, pc, opcode); break;
+		case 10: flags = op1010(stream, pc, opcode); break;
+		case 11: flags = op1011(stream, pc, opcode); break;
+		case 12: flags = op1100(stream, pc, opcode); break;
+		case 13: flags = op1101(stream, pc, opcode); break;
+		case 14: flags = op1110(stream, pc, opcode); break;
+		default: flags = op1111(stream, pc, opcode); break;
```

Comment:
The MAME coding style guidelines don't actually state what indentation level case labels use, and there remains no general consensus on this point in the wider world of programming, let alone other extant parts of the MAME codebase. Nevertheless, I'll move the indentation back here.